services/QuillLMS/app/controllers/blog_posts_controller.rb

```diff
@@ -10,12 +10,14 @@ class BlogPostsController < ApplicationController
 
 
   def index
+    @defer_js = true
```

Comment:
Since the same line is duplicated for so many routes, it would probably be cleanest to just pull it into a `before_action`.